content/guides/use-case/genai-pdf-bot/containerize.md

```diff
@@ -0,0 +1,120 @@
+
+The sample application used in this guide is a modified version of the PDF Reader application from the [GenAI Stack](https://github.com/docker/genai-stack) demo applications. The application is a full stack Python application that lets you ask questions about a PDF file.
+
+The application uses [LangChain](https://www.langchain.com/) for orchestration, [Streamlit](https://streamlit.io/) for the UI, [Ollama](https://ollama.ai/) to run the LLM, and [Neo4j](https://neo4j.com/) to store vectors.
+
+Clone the sample application. Open a terminal, change directory to a directory that you want to work in, and run the following command to clone the repository:
+
+```console
+$ git clone https://github.com/docker/docker-genai-sample
+```
+
+You should now have the following files in your `docker-genai-sample` directory.
+
+```text
+├── docker-genai-sample/
+│ ├── .gitignore
+│ ├── app.py
+│ ├── chains.py
+│ ├── env.example
+│ ├── requirements.txt
+│ ├── util.py
+│ ├── LICENSE
+│ └── README.md
+```
+
+## Initialize Docker assets
+
+Now that you have an application, you can use `docker init` to create the necessary Docker assets to containerize your application. Inside the `docker-genai-sample` directory, run the `docker init` command. `docker init` provides some default configuration, but you'll need to answer a few questions about your application. For example, this application uses Streamlit to run. Refer to the following `docker init` example and use the same answers for your prompts.
+
+```console
+$ docker init
+Welcome to the Docker Init CLI!
+
+This utility will walk you through creating the following files with sensible defaults for your project:
+  - .dockerignore
+  - Dockerfile
+  - compose.yaml
+  - README.Docker.md
+
+Let's get started!
+
+? What application platform does your project use? Python
+? What version of Python do you want to use? 3.11.4
+? What port do you want your app to listen on? 8000
+? What is the command to run your app? streamlit run app.py --server.address=0.0.0.0 --server.port=8000
+```
+
+You should now have the following contents in your `docker-genai-sample`
+directory.
+
+```text
+├── docker-genai-sample/
+│ ├── .dockerignore
+│ ├── .gitignore
+│ ├── app.py
+│ ├── chains.py
+│ ├── compose.yaml
+│ ├── env.example
+│ ├── requirements.txt
+│ ├── util.py
+│ ├── Dockerfile
+│ ├── LICENSE
+│ ├── README.Docker.md
+│ └── README.md
+```
+
+To learn more about the files that `docker init` added, see the following:
+ - [Dockerfile](../../../engine/reference/builder.md)
+ - [.dockerignore](../../../engine/reference/builder.md#dockerignore-file)
+ - [compose.yaml](../../../compose/compose-file/_index.md)
+
+
+## Run the application
+
+Inside the `docker-genai-sample` directory, run the following command in a
+terminal.
+
+```console
+$ docker compose up --build
+```
+
+Open a browser and view the application at [http://localhost:8000](http://localhost:8000). You should see a simple Streamlit application.
```

Comment:
Not sure if we want to point out here that starting the service may take a little while depending on your network connection, because it involves downloading quite a few dependencies.